src/citra_qt/bootmanager.cpp

```diff
@@ -484,3 +494,32 @@ void GLContext::MakeCurrent() {
 void GLContext::DoneCurrent() {
     context->doneCurrent();
 }
+
+void GRenderWindow::ConfineMouse() {
+    Layout::FramebufferLayout var = GetFramebufferLayout();
+    var.bottom_screen.left += GetMainWindow()->geometry().x();
+    var.bottom_screen.top += GetMainWindow()->geometry().y();
+    var.bottom_screen.right += GetMainWindow()->geometry().x();
+    var.bottom_screen.bottom += GetMainWindow()->geometry().y();
+    if (!GetMainWindow()->isFullScreen()) {
+        var.bottom_screen.bottom += point.y();
+        var.bottom_screen.top += point.y();
+    }
+#ifdef _WIN32
+    RECT rectangle;
+    rectangle.left = var.bottom_screen.left;
+    rectangle.right = var.bottom_screen.right;
+    rectangle.bottom = var.bottom_screen.bottom;
+    rectangle.top = var.bottom_screen.top;
+    layoutoption = Settings::values.layout_option;
+    confined = ClipCursor(&rectangle);
+#endif // _WIN32
+}
```

Comment:
```suggestion
void GRenderWindow::ConfineMouse() {
    Layout::FramebufferLayout var = GetFramebufferLayout();
#ifdef _WIN32
    RECT rectangle;
    rectangle.left = child_widget->mapToGlobal({static_cast<int>(var.bottom_screen.left), 0}).x();
    rectangle.right = child_widget->mapToGlobal({static_cast<int> (var.bottom_screen.right), 0}).x();
    rectangle.bottom =
        child_widget->mapToGlobal({0, static_cast<int> (var.bottom_screen.bottom)}).y();
    rectangle.top = child_widget->mapToGlobal({0, static_cast<int> (var.bottom_screen.top)}).y();
    layoutoption = Settings::values.layout_option;
    confined = ClipCursor(&rectangle);
#endif // _WIN32
}
```

This can likely be simplified further but i'll let you figure that out :)